### PR TITLE
Reenables eclipse station. Depends on #11937

### PR DESCRIPTION
**Pull request by alexkar598**
Depends on #11937

# Github documenting your Pull Request

Eclipse was disabled by me due to crashes, #11937 should fix that so it should be safe to bring back eclipse. Needs to be test merged first.

# Wiki Documentation

Eclipsestation is reenabled

# Changelog

:cl:  
rscadd: Eclipse station has been reenabled
/:cl:

### CODE REVIEW

**Comment by N3D6** [comment]
BASED

**Comment by Xoxeyos** [comment]
[https://youtu.be/P3ALwKeSEYs](url)

**Comment by Xoxeyos** [comment]
https://youtu.be/P3ALwKeSEYs

**Comment by Xoxeyos** [comment]
https://www.youtube.com/watch?v=P3ALwKeSEYs

**Comment by MayhemSailor** [comment]
Has eclipse been actually maintained in the last year or has it been ignored in regular updating?

**Comment by maxion12345** [comment]
yes it was disabled because of the pads

not because it was just so fucking massive you saw maybe three people a game even at high pop

**Comment by MayhemSailor** [comment]
> yes it was disabled because of the pads
> 
> not because it was just so fucking massive you saw maybe three people a game even at high pop

So without a shadow of a doubt your saying that from Oct 6th 2020 to today that eclipse has been actively maintained?

**Comment by MayhemSailor** [comment]
Personally I think eclipse should not be reenabled but permanently removed from the map list of Yogstation. There is no evidence that the community enjoys the map, and the admin team has repeatedly spread disinformation about the map and forced unpopular game modes whenever the map as slotted into rotation (possibly reinforcing the community dislike of the map). 

06848DA2-AF2C-4E2B-8C93-548B28F2CFB7

**Comment by JamieD1** [comment]
I dont understand, you moaned at me for keeping omega, which objectively is better and will get alot more playtime due to being a low pop map, then you want to enable a map which will be played maybe like once or twice a month at most.

I just dont understand.

**Comment by alexkar598** [comment]
Because omega is garbage. Eclipse was disabled because of lag, the other pr should fix the lag so the original reason to disable this is gone.Makes sense to reenable it now.

Omega was different as its:
1) Garbage
2) The same map as kilo except less liked

People hate delta and eclipse basically the same so it doesnt matter, and we do need a high pop map on occasion.

**Comment by patpol4** [comment]
Not sure how i feel about bringing it back. Always thought about it as more of a backup map that just sits there to accomodate tubertide pop if it happens.
When it comes to maintaining it, it was not being actively maintained at all in the past year but also it wasnt completely overlooked by me in reviews. It's just hard to develop and fix issues on a map that was played like 25 times in total.

**Comment by Moja10** [comment]
I love this Yogs unique map, it does need changes that aren't related to maintenance of the map itself, but I would love to force it during high pop for people to figure out what exactly are the addressable issues. I think it is totally unique and well designed.

**Comment by JamieD1** [comment]
> Because omega is garbage. Eclipse was disabled because of lag, the other pr should fix the lag so the original reason to disable this is gone.Makes sense to reenable it now.
> 
> Omega was different as its:
> 
> 1. Garbage
> 2. The same map as kilo except less liked
> 
> People hate delta and eclipse basically the same so it doesnt matter, and we do need a high pop map on occasion.

I mean thats your opinion no?

You have both stated "people" dont like it.

**Comment by maxion12345** [comment]
eclipse is simply far too massive far our currently population. It makes playing it boring. This might have been playable during the 90 player tides, our current "maybe 60-70 players once a week if the stars align" simply means eclipse doesn't have a place in yogstation anymore.

tl;dr
making a station so large you need teleporters was bound to fail

**Comment by N3D6** [comment]
> Personally I think eclipse should not be reenabled but permanently removed from the map list of Yogstation. There is no evidence that the community enjoys the map, and the admin team has repeatedly spread disinformation about the map and forced unpopular game modes whenever the map as slotted into rotation (possibly reinforcing the community dislike of the map).
> 
> 06848DA2-AF2C-4E2B-8C93-548B28F2CFB7

i won't deny i used to really dislike eclipse but its much better than delta and we are hurting for map variety at this point

**Comment by maxion12345** [comment]
what if i made boxstation but bigger

**Comment by alexkar598** [comment]
> 
> 
> what if i made boxstation but bigger

Welcome to metastation, enjoy your stay

**Comment by adamsong** [comment]
> hurting for map variety

Doesn't matter how many maps we have if "Map randomly rotating to \<not box\>" gets immediately followed up by "Map vote, Box wins"
